tests/chalice/test_views.py

```diff
@@ -94,3 +94,11 @@ def test_graphiql_query_with_no_request_body():
         response = client.http.post("/graphql", headers=headers, body="")
         assert response.status_code == HTTPStatus.OK
         assert response_is_of_error_type(response)
+
+
+def test_deprecated_render_graphiql():
+    with Client(app) as client:
+        headers = {"Accept": "application/json"}
+        response = client.http.get("/deprecated-graphql", headers=headers)
+        assert response.status_code == HTTPStatus.OK
+        assert response_is_of_error_type(response)
```

Comment:
can we assert that we trigger the warning?

Reply:
Yes I would like to do that, but I wasn't able to catch the warning... I [tried this approach](https://stackoverflow.com/a/3892301/4329653) but wasn't able to capture anything... 
Maybe is because it's an API call, so anything happening "on the other side" is quite difficult to capture? 
I will investigate a bit more on this...

Reply:
I added a unit test on the view to test that is throwing the warning :)